sqlserver/assets/configuration/spec.yaml

```
@@ -86,13 +86,15 @@ files:
       value:
         type: boolean
         example: false
-    - name: object_name
+    - name: db_frag_object_names
```

```suggestion
    - name: db_fragmentation_object_names
```

Since we spell it out in the other option, should be consistent here too.